src/types.ts

```diff
@@ -1,9 +1,37 @@
-export type AuthObject = {
-  email: string;
-  password: string;
-}
+import { CurrentAdmin } from 'admin-bro'
+
+/**
+ * @memberof module:@admin-bro/koa
+ * @alias KoaAuthenticateFunction
+ *
+ * @description
+ * Authentication function
+ */
+export type KoaAuthenticateFunction = (
```

Comment:
I had to put this to a type because otherwise, I wouldn't be able to document it